heroku local not functioning

I was going through the "get started with python" tutorial and trying to run the app locally, 
I typed

   `heroku local web -f Procfile.windows`

And I got the error:

```
Error: Cannot find module '../../../src/load-foreman-procfile'
    at Object.<anonymous> (C:/Program Files/heroku/client/node_modules/@heroku-cli/plugin-local/lib/commands/local/index
.js:5:18)
```

I tried uninstalling the heroku cli, redownloading the installer and reinstalling; but I am still getting the same error message. 

The output of `heroku local:version` is:

```
internal/modules/cjs/loader.js:670
    throw err;
    ^

Error: Cannot find module 'C:\Program Files\heroku\client\node_modules\@heroku-cli\plugin-local\lib\run-foreman.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:668:15)
    at Function.Module._load (internal/modules/cjs/loader.js:591:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:877:12)
    at internal/main/run_main_module.js:21:11
```

The output of `heroku plugins --core` is:

```
@oclif/plugin-commands 1.2.2 (core)
@oclif/plugin-help 2.1.6 (core)
@oclif/plugin-legacy 1.1.4 (core)
@oclif/plugin-not-found 1.2.2 (core)
@oclif/plugin-plugins 1.7.7 (core)
@oclif/plugin-update 1.3.9 (core)
@oclif/plugin-warn-if-update-available 1.7.0 (core)
@oclif/plugin-which 1.0.3 (core)
addons-v5 7.24.0 (core)
apps-v5 7.26.0 (core)
auth 7.24.0 (core)
autocomplete 7.24.0 (core)
buildpacks 7.24.0 (core)
certs 7.24.0 (core)
certs-v5 7.24.0 (core)
ci 7.24.3 (core)
ci-v5 7.24.0 (core)
config 7.24.0 (core)
container-registry-v5 7.24.4 (core)
git 7.24.0 (core)
heroku 7.26.0 (core)
local 7.26.0 (core)
oauth-v5 7.24.0 (core)
orgs-v5 7.26.0 (core)
pg-v5 7.25.0 (core)
pipelines-v5 7.26.0 (core)
ps 7.24.0 (core)
ps-exec 2.3.5 (core)
redis-v5 7.25.0 (core)
run-v5 7.24.0 (core)
spaces 7.26.0 (core)
status 7.24.0 (core)
webhooks-v5 7.24.0 (core)
```

Neither load-foreman-procfile nor run-foreman.js exist on my system. I am running heroku 7.26.0 64-bit on Windows 7 64-bit.

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
   },
   "private": true,
   "scripts": {
-    "test": "lerna run test",
+    "test": "lerna run test --concurrency 1",
     "version": "cp packages/cli/CHANGELOG.md CHANGELOG.md && git add CHANGELOG.md",
     "posttest": "standard"
   },

--- a/packages/local/src/commands/local/index.ts
+++ b/packages/local/src/commands/local/index.ts
@@ -1,7 +1,7 @@
 import {FileCompletion} from '@heroku-cli/command/lib/completions'
 import {Command, flags} from '@oclif/command'
 
-const Procfile: any = require('../../../src/load-foreman-procfile')
+const Procfile: any = require('../../load-foreman-procfile')
 
 import {fork as foreman} from '../../fork-foreman'
 

